src/App.js

```diff
@@ -70,9 +74,15 @@ class PageFetchWrapper extends Component {
 
     render() {
         return (
-            <Layout>
-                {this.state.error === ERROR_UNAUTHORIZED_USER ? <NoAuth /> : <Page data={this.state.layout} />}
-            </Layout>
+            <PageContext.Provider
```

Comment:
I need to pass the mode of the page all the down the tree (is used not to show some things in live mode) so instead of pass it as params all the way down I use the [React Context API](https://reactjs.org/docs/context.html).